src/main/scala/org/broadinstitute/hail/expr/Type.scala

```diff
@@ -657,7 +657,7 @@ case class TStruct(fields: IndexedSeq[Field]) extends Type {
     if (size == 0)
       Gen.const(Annotation.empty)
     else
-      Gen.getSize.flatMap(fuel =>
+      Gen.size.flatMap(fuel =>
```

Comment:
Ah yeah I suppose so. I was waffling on whether or not to make this a `for`. The notation is .. eh, especially because the word `for` really gives the wrong impression:

```scala
for (fuel <- Gen.size) yield {
  if (size < fuel) Gen.const(Annotation.empty)
  else Gen.uniformSequence(fields.map(f => f.`type`.genValue)).map(a => Annotation(a: _*)))
}
```